### ISSUE TITLE
Feature: hyperlink between pages

### ISSUE THREAD

**Issue by MichaelCurrin** [ShailenNaidoo/hydrogen#14]
Add instructions (or functionality if it doesn't exist yet) to link in the JS between pages. e.g. How to link _to_ page `a.html` or `b/c.html` _from_ within `index.html`.

jekyll uses a page system for the `link` function, so you have to just know the path to the `.md` file in the project and it will figure out the HTML path for you.

**Comment by ShailenNaidoo** [comment]
@MichaelCurrin Can you provide a bit more context about the feature? Do you mean that if I reference the file `/docs/index.js` Hydrogen would automatically infer that as `/docs/index.html`?

**Comment by MichaelCurrin** [comment]
Correct. One should only provide the path to the file (this relative to project root) and the function should figure what the browser path should be.
